File: JS/ContandoNumerosPares.test.js
/**
 * @version JAVASCRIPT
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ node NomeDaClasse.test
 */
const TEST = require('tape')
const OBJ = require('./ContandoNumerosPares.js')

TEST('Starting ContandoNumerosPares test...', (t) => {
    //Test 1
    let want = 2
    let got = OBJ.countEven( 4 ) 
    t.assert( got === want, "Expect: "+want+" - the return was: "+got)
    //Test 2
    want = 14
    got = OBJ.countEven( 30 )
    t.assert( got === want, "Expect: "+want+" - the return was: "+got)

    t.end()
})

File: JAVA/ContandoNumerosParesTest.java
/** 
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ mvn clean test -Dtest=your.package.TestClassName
 */ 
package JAVA;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class ContandoNumerosParesTest {

	@Test
	public void testContandoNumerosPares() throws Exception {
		//Test 1
		int want = 2;
		int got = ContandoNumerosPares.countEven(4);
		assertEquals(want, got);
		//Test 2
		want = 14;
		got = ContandoNumerosPares.countEven(30);
		assertEquals(want, got);
	}

}
